### PR TITLE
Win2022 azure edition core alias

### PR DESCRIPTION
**Pull request by mgreenegit**
## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute/ChangeLog.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
 * Added `-InstanceView` parameter to `Get-AzRestorePoint`
 * Added parameter `-ScriptString` to `Invoke-AzvmRunCommand` and `Invoke-AzvmssRunCommand`
 * Added parameter `-ScaleInPolicyForceDeletion` to `Update-Azvmss`
+* Added image alias 'Win2022AzureEditionCore'
```
Comment (VeryEarly) [inline src/Compute/Compute/ChangeLog.md:41]: please move changelog under line 22

**Comment by VeryEarly** [comment]
Hi @Sandido @grizzlytheodore ,

Please help verify this change

**Comment by grizzlytheodore** [comment]
@VeryEarly  LGTM. no customer impact

**Review by grizzlytheodore — APPROVED** [review]

**Comment by isra-fel** [comment]
There was an issue with our CI system and we are working on it.

**Comment by isra-fel** [comment]
@VeryEarly  Hi Yabo, the CI issue has been fixed. Please go ahead with this PR. Thanks.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**